src/index.ts

```diff
@@ -162,6 +165,28 @@ async function main() {
   } catch (exception) {
     logger.error({ exception }, 'BlockchainReader was unable to start')
   }
+
+  return {
+    stop: async () => {
+      logger.info('Po.et node terminating...')
+
+      await api.stop()
```

Comment:
What happens if one `.close` fails?  I think you should catch this error, in the each function

Reply:
Then the whole app exits anyway. What would be the purpose of catch()ing?

Reply:
but, it's right that some service was stopped but other services no... and after that crash the app?